angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -177,8 +177,16 @@
         <div class="sub-container">
```

Comment:
Playing with the action menu in storybook I'm finding that there is a lag for opening the action menu proportional to the size of the data set assigned to the table. If I bump the large data set size up to 100k elements it scrolls fine but opening the action menu then takes a few seconds.

Reply:
That's a storybook issue related to it listening to events and then serializing the event (including the entire table state since the table is the event's `target`). Setting `handles` to an empty array in the story makes the delay go away.

I thought it was better to have the story handle those events as documentation that those events exist, but maybe it would be better to explicitly document them and remove the storybook handlers?

Reply:
lol yikes. Yea lets document it in the storybook controls similar to the banner (good to either way).

Seems reasonable to disable the actions panel if it's performing so poorly. Alternatively, it looks like we can configure it to log less: https://github.com/storybookjs/storybook/blob/master/addons/actions/ADVANCED.md#configuration

Maybe we should set it to a depth of 1 or something?

Reply:
Got curious and played around with it in preview.js with a maxDepth of 2 and hit: https://github.com/storybookjs/storybook/issues/17409#issuecomment-1303773808

But using a maxDepth of 6 seems to prevent the perf issue while giving a good amount of the data to inspect:
![image](https://user-images.githubusercontent.com/1588923/219758915-41413d0e-9c0a-4731-b1e5-68bb9529ba03.png)

Reply:
I was actually in the process of responding to say that 6 seems to be the magic number to get most of the data we want while avoiding performance problems.

Reply:
I'd say lets go ahead and do it in preview.js to prevent the issue popping up globally (I think the change in the table story technically does impact it globally but might be intermittent based on which story you load first compared to doing it in preview.js)